client/src/util.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
```

Comment:
There's currently a `utils.ts` file under `components/` directory. This introduces a question about the positioning for the 2 utility files. I guess the SAS code utility will be larger moving forward thus deserves a separate file.
So how about name it something like `sasCodeUtil.ts` and put it into the `components/` directory?

Reply:
Good point. Sure, I'll move it into components/ directory. What about if we name it "utils_sasCode.ts", so that any "utils*" files will appear adjacent in explorer?

Reply:
I moved and renamed it to components/utils_sasCode.ts.